### PR TITLE
Added testdriver automation for html/editing/focus/focus-02.html

### PR DESCRIPTION
**Pull request by kereliuk**
Automate all the things!

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/26866)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/362272654)

File: html/editing/focus/focus-02.html [hunk 1]
```diff
@@ -7,12 +7,8 @@
 <meta assert="assert" content="Check if the key events received by document are targeted at the element when no element is focused">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<h2>Steps:</h2>
```
Comment (jgraham) [inline html/editing/focus/focus-02.html]: Do we have some convention about leaving manual steps in here so that they can be run without testdriver?

Comment (foolip) [reply html/editing/focus/focus-02.html]: Not really. I think leaving the alone where they exist would be OK, but I wouldn't expect the instructions to stay working over time, or that they'd be added for new tests, so I'd perhaps slightly prefer removing them. WDYT?

**Review by foolip — APPROVED** [review]

File: html/editing/focus/focus-02.html (hunk 1 above)
Comment (kereliuk) [reply html/editing/focus/focus-02.html]: I think they should be removed.
When new tests are added with testdriver they generally aren't accompanied by manual steps incase they are run without testdriver, so I think we should stick to this convention. I also agree with what @foolip says with maintenance, but @jgraham you bring up a good point that makes me wonder whether we should even allow testdriver tests to be run in any other way than wpt run. I opened #10330 to further address this.

**Pull request merged**